src/rules/rulepanel.ts

```diff
@@ -111,6 +111,18 @@ function renderImpact(softwareQuality: string, severity: string, resolver: Resou
 </div>`;
 }
 
+function renderStandardModeSeverityDetails(ruleType: string, severity: string, resolver: ResourceResolver) {
+  const ruleTypeToLowerCase = ruleType.toLocaleLowerCase('en-us');
+  const severityToLowerCase = severity.toLocaleLowerCase('en-us');
+  const ruleTypeImgSrc = resolver.resolve('images', 'standardMode', ruleTypeToLowerCase, `${severityToLowerCase}.svg`);
+  const severityImgSrc = resolver.resolve('images', 'severity', `${severityToLowerCase}.svg`);
+  return `<div class="impact severity-${severityToLowerCase} capsule">
+  <img alt="${clean(ruleType)}" src="${ruleTypeImgSrc}"/>
+  &nbsp;${clean(ruleType)}&nbsp;
+  <img alt="${severityToLowerCase}" src="${severityImgSrc}"/>
```

Comment:
Could we imagine using the contents of the SVG files as elements that could be styled with CSS instead of having 15 icons to account for all severities?